apps/packages-playground/src/App.tsx

```diff
@@ -0,0 +1,33 @@
+import "./App.css";
```

Comment:
@ahmadshaheer : Do we require packages playground as part of this repo ?

Reply:
@YounixM, IMO, it would be better to have it, in case we want to test any components/packages while developing.